Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Flow.Launcher.Plugin.Explorer.Search.QuickAccessLinks
+{
+    public class QuickAccess
+    {
+        private readonly ResultManager resultManager;
+
+        public QuickAccess(PluginInitContext context)
+        {
+            resultManager = new ResultManager(context);
+        }
+
+        internal List<Result> AccessLinkListMatched(Query query, List<AccessLink> accessLinks)
+        {
+            if (string.IsNullOrEmpty(query.Search))
+                return new List<Result>();
+
+            string search = query.Search.ToLower();
+
+            var queriedAccessLinks =
+                accessLinks.Where(x => x.Nickname.StartsWith(search, StringComparison.OrdinalIgnoreCase));
+
+            return queriedAccessLinks
```

Comment:
Maybe using an OrderBy ResultType and ThenBy Title will be more clear? since ResultType is an enum and it should be comparable.